src/main/java/seedu/allonus/expense/ExpenseParser.java

```diff
@@ -15,6 +15,7 @@
     public static final int LEFT_SIDE = 0;
     public static final int ZERO = 0;
     public static final int INDEX_TO_BE_DELETED = 1;
+    public static final int INDEX_TO_BE_EDITED = 1;
```

Comment:
I think these two constants serve a very similar purpose? If I understand correctly, would this break down if you're indexing an entry that goes into the double digits?

Reply:
Thank you for the comment. I don't think it will break down as this integer constant is purely to access the 2nd element after splitting the user's input into 2.

Reply:
Ah my bad, that makes sense! But maybe it's still worth merging the two into something like `INDEX_TO_BE_PARSED` that can be used for both features?

Reply:
Okay sounds good! Edited accordingly.